timesketch/lib/tasks.py

```diff
@@ -181,37 +186,78 @@ def build_index_pipeline(file_path, timeline_name, index_name, file_extension,
     return chain(index_task, index_analyzer_chain)
 
 
-def build_sketch_analysis_pipeline(sketch_id):
+def build_sketch_analysis_pipeline(sketch_id, searchindex_id, user_id,
+                                   analyzer_names=None, analyzer_kwargs=None):
     """Build a pipeline for sketch analysis.
 
     Args:
-        sketch_id: The ID of the sketch to analyze.
+        sketch_id (int): The ID of the sketch to analyze.
+        searchindex_id (int): The ID of the searchindex to analyze.
+        user_id (int): The ID of the user who started the analyzer.
+        analyzer_names (list): List of analyzers to run.
+        analyzer_kwargs (dict): Arguments to the analyzers.
 
     Returns:
         Celery group with analysis tasks or None if no analyzers are enabled.
     """
     tasks = []
+    auto_analyzers = current_app.config.get('AUTO_SKETCH_ANALYZERS', None)
 
-    # Exit early if sketch analyzers are disabled.
-    if not current_app.config.get('ENABLE_SKETCH_ANALYZERS', False):
+    # Exit early if no sketch analyzers are configured to run.
+    if not (analyzer_names or auto_analyzers):
         return None
```

Comment:
no need to define None....

`return` does return None

Reply:
I know, linter complains though because there are other returns in this function. I rather be explicit here than ignoring linter.